leshan-client-cf/src/main/java/org/eclipse/leshan/client/californium/CaConstraintCertificateVerifier.java

```diff
@@ -46,60 +48,51 @@
  *       certificate usage allows both trust anchors and CA certificates,
  *       the certificate might or might not have the basicConstraints
  *       extension present.
- *
+ * </pre>
```

Comment:
I need to add <pre> or formatter change this.

Reply:
Ok. I assume the pre tags ;)